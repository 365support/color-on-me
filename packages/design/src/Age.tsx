import { css } from '@emotion/react';

export const Age = () => {
  return (
    <div
      css={css`
        width: 200px;
        height: 200px;
        background-color: red;
      `}
    >
      hello
    </div>
  );
};
