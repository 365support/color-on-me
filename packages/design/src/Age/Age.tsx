import { css } from '@emotion/react';

function Age() {
  return (
    <div
      css={css`
        width: 200px;
        height: 200px;
        background-color: red;
      `}
    >
      hello
    </div>
  );
}

export default Age;
