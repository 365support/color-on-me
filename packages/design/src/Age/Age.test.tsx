import renderer from 'react-test-renderer';
import Age from './Age';

it('changes the class when hovered', () => {
  const component = renderer.create(<Age />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // re-rendering
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
