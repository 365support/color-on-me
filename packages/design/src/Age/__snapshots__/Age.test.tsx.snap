// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
[
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-13xn979-Age{width:200px;height:200px;background-color:red;}",
      }
    }
    data-emotion="css 13xn979-Age"
  />,
  <div
    className="css-13xn979-Age"
  >
    hello
  </div>,
]
`;

exports[`changes the class when hovered 2`] = `
[
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-13xn979-Age{width:200px;height:200px;background-color:red;}",
      }
    }
    data-emotion="css 13xn979-Age"
  />,
  <div
    className="css-13xn979-Age"
  >
    hello
  </div>,
]
`;
